docs/kyma/04-12-details.md

```diff
@@ -5,12 +5,26 @@ type: Installation
 
 This section describes in details some of the installation mechanisms.
 
```

Comment:
I would rename the doc to `Error handling` If we have more items to add to this one we can then think of changing the name. Error handling says more than `details`.
Rename the title of the doc in the metadata and the name of the file.

Reply:
done!